### DISCUSSION TITLE
Add links to stack trace entries

### DISCUSSION THREAD

**Discussion by manuelmeurer** [getsentry/sentry#34686] [Product Spitballin']
Hey hey! 👋 

I noticed there is no way to add links to stack trace entries in order to open the file in my editor, so I built a small Chrome extension that does this: https://github.com/manuelmeurer/sentry-stack-trace-links

It’s pretty essential to me to open the files referenced in the stack trace, not just see the few surrounding lines.

But maybe I missed another way to do this? 😄 

If not, are there any plans to add this feature to Sentry in the future?

**Comment by chadwhitacre** [comment]
Thanks for sharing @manuelmeurer! I believe we have stacktrace linking over into GitHub, @ashwoods @getsentry/owners-native do we have docs on that? Thoughts on linking elsewhere?

**Reply by manuelmeurer** [reply]
Yeah, I have linking to Github set up too, but in 99% of the cases I want to jump directly into my editor instead of Github.
It would be great if Sentry allowed flexibly linking anywhere, i.e. as a user I could set some kind of URL template, which would be `vscode://file${localPath}${filename}:${line}` for VS Code, `https://github.com/${repo}/blob/${branch}/${filename}#${line}` for Github, etc.
